gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.5.1-all.zip
```

Comment:
Why using a old version of gradle? The 4.7 is just available!

Reply:
Yes but no: Anyone else (using Gradle 4.6+) getting the follo...

https://www.reddit.com/r/androiddev/comments/8ed6d3/android_studio_312_bugfix_update_now_available/dxueu6j?utm_source=reddit-android